test/evergreen_develop.yml

```diff
@@ -3,34 +3,16 @@
 include:
 - filename: test/evergreen.yml
 
-
 # Build variants that need to be run only on the develop branch.
 buildvariants:
-- name: ubuntu2004-perf-tests
-  display_name: Ubuntu 20.04 Performance tests
+# Template for perf-tasks
+- &perf-tasks-template
+  name: ubuntu2004-perf-tests
```

Comment:
Is it clearer to include "template" in the name, preferably matching the name in the line above?

Reply:
I tried that, but for some reason it gives me this error when I try to create a patch build.
```
Unexpected reply from server (500 Internal Server Error): 400 (Bad Request): invalid patched project config: invalid patched config syntax: buildvariant 'ubuntu2004-perf-template' must either specify run_on field or have every task specify run_on
buildvariant 'ubuntu2004-perf-template' does not have a display name
```

Reply:
Let me give a try to see if I can make it work :)